PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.cxx

```diff
@@ -513,13 +513,14 @@ AliAnalysisTaskJetExtractor::AliAnalysisTaskJetExtractor() :
   fTruthMaxLabel(100000),
   fHadronMatchingRadius(0.4),
   fJetMatchingRadius(0.3),
+  fSharedPtFraction(0.5),
```

Comment:
I think fJetMatchingSharedPtFraction would be a more informative name here and it would fit to fJetMatchingRadius

Reply:
Resolved.